pack_and_terraform.bash

```diff
@@ -79,78 +86,79 @@ if echo "$RUNTF" | grep -iq "^y"; then
         fi
-	sed -i '' -e "s/proxy_ami=\"\"/proxy_ami=\"$PROXY_AMI\"/g" ../tf_variables
 
 	if [ -z "$CHOSTNAME" ]; then
 		read -p "Enter your hostname name like www.example.com: " CHOSTNAME
 	fi
-	sed -i '' -e "s#hostname=\"www.example.com\"#hostname=\"$CHOSTNAME\"#g" ../tf_variables
 
 	if [ -z "$BUCKET" ]; then
 		read -p "Enter your desired kube bucket name: " BUCKET
 	fi
-	sed -i '' -e "s/kube_bucket=\"\"/kube_bucket=\"$BUCKET\"/g" ../tf_variables
 
-	sed -i '' -e "s#kube_ssh_key=\"ssh-rsa XXXX\"#kube_ssh_key=\"$SSHKEY\"#g" ../tf_variables
 
 	ADDSSHKEY=`curl -s https://github.com/philloooo.keys | tail -1`
 	echo "Phillis' key is: $ADDSSHKEY"
-	sed -i '' -e "s#kube_additional_keys=\"\"#kube_additional_keys=\"- \\\\\"$ADDSSHKEY\\\\\"\\\\n\"#g" ../tf_variables
 
 	if [ -z "$CLIENTID" ]; then
 		read -p "Enter your Google OAuth2 Client ID: " CLIENTID
 	fi
-        sed -i '' -e "s/google_client_id=\"\"/google_client_id=\"$CLIENTID\"/g" ../tf_variables
 
 	if [ -z "$CLIENTSECRET" ]; then
 		read -p "Enter your Google OAuth2 Client Secret: " CLIENTSECRET
 	fi
-        sed -i '' -e "s/google_client_secret=\"\"/google_client_secret=\"$CLIENTSECRET\"/g" ../tf_variables
 
 	export LC_CTYPE=C
 	HMAC=`cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1 | base64`
 	echo "Your HMAC encryption key is: $HMAC"
-	sed -i '' -e "s/hmac_encryption_key=\"\"/hmac_encryption_key=\"$HMAC\"/g" ../tf_variables
 
 	USERAPIDBPASS=`cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1`
         echo "Your User API DB password is: $USERAPIDBPASS"
 	GDCAPIDBPASS=`cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1`
         echo "Your GDC API DB password is: $GDCAPIDBPASS"
 	INDEXDDBPASS=`cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1`
         echo "Your IndexD DB password is: $INDEXDDBPASS"
-	sed -i '' -e "s/db_password_userapi=\"\"/db_password_userapi=\"$USERAPIDBPASS\"/g" ../tf_variables
-	sed -i '' -e "s/db_password_gdcapi=\"\"/db_password_gdcapi=\"$GDCAPIDBPASS\"/g" ../tf_variables
-	sed -i '' -e "s/db_password_indexd=\"\"/db_password_indexd=\"$INDEXDDBPASS\"/g" ../tf_variables
-
 	INDEXD=`cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1`
         echo "Your indexd write password is: $INDEXD"
-	sed -i '' -e "s/gdcapi_indexd_password=\"\"/gdcapi_indexd_password=\"$INDEXD\"/g" ../tf_variables
 
+    export AWS_ACCESS_KEY=$AWS_ACCESS_KEY
+    export AWS_SECRET_KEY=$AWS_SECRET_KEY
+    export VPC_NAME=$VPC_NAME
+    export LOGIN_AMI=$CLIENT_AMI
+    export PROXY_AMI=$PROXY_AMI
+    export BASE_AMI=$SOURCE_AMI
+    export CHOSTNAME=$CHOSTNAME
+    export SSHKEY=$SSHKEY
+    export ADDSSHKEY=$SSHKEY
+    export BUCKET=$BUCKET
+    export CLIENTSECRET=$CLIENTSECRET
+    export CLIENTID=$CLIENTID
+    export HMAC=$HMAC
+    export USERAPIDBPASS=$USERAPIDBPASS
+    export INDEXDDBPASS=$INDEXDDBPASS
+    export GDCAPIDBPASS=$GDCAPIDBPASS
+    export INDEXD=$INDEXD
```

Comment:
Can we pass these just to envsubst so we don't pollute the env variables globally?